README.md

```diff
@@ -1,96 +1,75 @@
 root-signing project maintains the TUF repository used to securely deliver the
 _Sigstore trust root (trusted_root.json)_ to Sigstore clients.
 
-> [!NOTE]
-> root-signing is currently undergoing a migration to new maintenance tooling, see
-> https://github.com/sigstore/root-signing/issues/929. Sigstore client operation
-> should not be disrupted but the information in this README may be momentarily
-> out of date.
+### Documentation
```

Comment:
The first section has no header, I think we should add one as it looks a bit odd reading the README now.